schema.json

```diff
@@ -29,7 +29,12 @@
     },
     "type": "object",
     "properties": {
+        "$id": {
+            "type": "string",
+            "format": "uriref"
+        },
         "id": {
+            "description": "This keyword has been deprecated in favor of \"$id\"",
```

Comment:
I think dropping from spec and optionally supporting in implementations addresses migration issues. I'd rather we keep it simple here, so new implementations don't have to carry this baggage...